Library/OcAudioLib/OcAudioVoiceOver.c

```diff
@@ -27,6 +27,56 @@
 
 #include "OcAudioInternal.h"
 
+//
+// Convert from Apple file id to file base name and type.
+//
+CONST APPLE_VOICE_OVER_FILE_MAP  gAppleVoiceOverAudioFiles[AppleVoiceOverAudioFileMax] = {
```

Comment:
To save space we should rather use either two tables for localised and not or subtract 0x30 from localised indices. I like the idea with two tables more.

Reply:
@vit9696 - Okay, I think remaining resolved.